docs/developers-guide/substreams-sink-files.md

```diff
@@ -0,0 +1,220 @@
+---
+description: StreamingFast Substreams sink files
+---
+
+# `substreams-sink-files` introduction
+
+## Outline
+
+1. Download code and tools
+2. Run code and verify output
+3. Substreams modifications
+4. Understanding the sink tool
+5. Run tool and verify output
+6. Conclusion and review
+
+**<i>NOTE: We can delete this outline later if you want, it helps me with development and creation of this content. If you would like to add or remove or restructure the main sections/steps of the document, let's do it here and then work down into the content.</i>**
+
+### Purpose
+
+This documentation exists to assist you in understanding and beginning to use the StreamingFast `substreams-sink-files` tool. The accompanying Substreams module is provided as a basic example of the elements required for sinking blockchain data into files-based storage solutions.
+
+### Overview
+
+The `substreams-sink-files` tool provides the ability to pipe data extracted from a blockchain to various types of files-based persistence solutions.
+
+For example, you could extract all of the ERC20, ERC721, and ERC1155 transfers from the Ethereum blockchain and persist the data to a files-based store.
+
+Substreams modules are created and prepared specifically for the sink tool. After the sink tool begins running, automated tasks can be setup to have [BigQuery](https://cloud.google.com/bigquery), or other files-based storage solutions, ingest the data.
+
+By using the automated ingestion tasks you can also route the data to [Clickhouse](https://clickhouse.com), custom scripts and other related data storage and processing tools accepting a file format. This can only be accomplished indirectly. It's possible to automate further ingestion from files to data stores.
```

Comment:
Should be merged together, last sentence talks about ingestion task to BigQuery, and we repeat the same on firdst sentence of next paragraph for Clickhouse which are similar tool.

Reply:
Merged.